### ISSUE TITLE
warmup or different lr for backbone and head when full finetuning

### ISSUE THREAD

**Issue by fedenanni** [alan-turing-institute/heartfm-evals#34]
We discussed yesterday whether we need warmup when full fine-tuning. Another option is what @rwood-97 suggested a few days back (different lr , in this case for backbone and head). I am happy with any of them or even with not doing anything and see how the different models behave, but I think we should just be consistent between segmentation and classification. Tell me which one you prefer, and then i'll make sure i'll do the same on the classification side.
